### ISSUE TITLE
Writing: Finalize dataset split image. 

### ISSUE THREAD

**Issue by srush** [bigscience-workshop/promptsource#458]
Need to know the exact splits for this figure. 

[datasets (2).pdf](https://github.com/bigscience-workshop/promptsource/files/7231667/datasets.2.pdf)
e.

**Comment by awebson** [comment]
If by "split" you meant task categories (the grey encirclement in your figure), please refer to the task_by_convention column in this [spreadsheet](https://docs.google.com/spreadsheets/d/1ELEM-BY5E-5Tqy34jguT91ChwAguoWfyDEN0OzOhjSk/edit#gid=1315886539), in which everything up to row 75 should be included in your figure. Thanks!

**Comment by srush** [comment]
Perfect. and `do_train` and `do_eval` refer to which side they are on?

**Comment by awebson** [comment]
Yup! With the caveat that they reflect the vanilla "D4 only" training mixture, which I will explain in the main paper.
